week_4/quiz-trivia-app/src/app/screens/finish-page/statistics/statistics.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import { QuizService } from '../../../data/quiz.service';
+import { Quiz } from '../../../data/quiz.model';
+import { StorageService } from '../../../storage.service';
+
+@Component({
+  selector: 'app-statistics',
+  standalone: true,
+  imports: [],
+  templateUrl: './statistics.component.html',
+  styleUrl: './statistics.component.css',
+})
+export class StatisticsComponent implements OnInit {
+  subject: Quiz | undefined = undefined;
+  totalScore!: number;
+  theme!: string;
+
+  constructor(
+    private quizService: QuizService,
+    private storedService: StorageService
+  ) {}
+
+  ngOnInit(): void {
+    this.quizService.subject$.subscribe((quiz) => (this.subject = quiz));
+    this.quizService.score$.subscribe((points) => (this.totalScore = points));
+
+    const storedScore = this.storedService.getData<number>('scorePoints');
```

Comment:
you dont need this two variables storedScore and storedSubject

Reply:
done